modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -144,6 +85,10 @@ export default {
   name: 'AposPiecesManager',
   mixins: [ AposTableMixin, AposModalParentMixin ],
   props: {
+    // TEMP From Table Mixin:
```

Comment:
All of these TEMP comments will be taken out by the final stage of this thread of work, when the relationships manager is set up to use all three of the other manager views.